rpc-server/src/metrics.rs

```diff
@@ -60,608 +68,133 @@ impl OptimisticUpdating {
+    pub(crate) static ref METHODS_CALLS_COUNTER: IntCounterVec = register_int_counter_vec(
+        "methods_calls_counter",
+        "Total number of calls to the method",
+        &["method_name"] // This declares a label named `method name`
     ).unwrap();
 
-    pub(crate) static ref QUERY_VIEW_ACCESS_KEY_ERROR_1: IntCounter = try_create_int_counter(
-        "query_view_access_key_error_1",
-        "Query.view_access_key error 1: ReadRPC success, NEAR RPC error"
-    ).unwrap();
-
-    pub(crate) static ref QUERY_VIEW_ACCESS_KEY_ERROR_2: IntCounter = try_create_int_counter(
-        "query_view_access_key_error_2",
-        "Query.view_access_key error 2: ReadRPC error, NEAR RPC success"
-    ).unwrap();
-
-    pub(crate) static ref QUERY_VIEW_ACCESS_KEY_ERROR_3: IntCounter = try_create_int_counter(
-        "query_view_access_key_error_3",
-        "Query.view_access_key error 3: ReadRPC error, NEAR RPC error"
-    ).unwrap();
-
-    pub(crate) static ref QUERY_VIEW_ACCESS_KEY_ERROR_4: IntCounter = try_create_int_counter(
-        "query_view_access_key_error_4",
-        "Query.view_access_key error 4: Failed to compare. Network or parsing error"
-    ).unwrap();
-    // end QUERY.view_access_key
-}
-
-lazy_static! {
-    // QUERY.view_state
-    pub(crate) static ref QUERY_VIEW_STATE_ERROR_0: IntCounter = try_create_int_counter(
-        "query_view_state_error_0",
-        "Query.view_state error 0: ReadRPC success, NEAR RPC success"
-    ).unwrap();
-
-    pub(crate) static ref QUERY_VIEW_STATE_ERROR_1: IntCounter = try_create_int_counter(
-        "query_view_state_error_1",
-        "Query.view_state error 1: ReadRPC success, NEAR RPC error"
-    ).unwrap();
-
-    pub(crate) static ref QUERY_VIEW_STATE_ERROR_2: IntCounter = try_create_int_counter(
-        "query_view_state_error_2",
-        "Query.view_state error 2: ReadRPC error, NEAR RPC success"
-    ).unwrap();
-
-    pub(crate) static ref QUERY_VIEW_STATE_ERROR_3: IntCounter = try_create_int_counter(
-        "query_view_state_error_3",
-        "Query.view_state error 3: ReadRPC error, NEAR RPC error"
-    ).unwrap();
-
-    pub(crate) static ref QUERY_VIEW_STATE_ERROR_4: IntCounter = try_create_int_counter(
-        "query_view_state_error_4",
-        "Query.view_state error 4: Failed to compare. Network or parsing error"
-    ).unwrap();
-    // end QUERY.view_state
-}
-
-lazy_static! {
-    // QUERY.function_call
-    pub(crate) static ref QUERY_FUNCTION_CALL_ERROR_0: IntCounter = try_create_int_counter(
-        "query_function_call_error_0",
-        "Query.function_call error 0: ReadRPC success, NEAR RPC success"
-    ).unwrap();
-
-    pub(crate) static ref QUERY_FUNCTION_CALL_ERROR_1: IntCounter = try_create_int_counter(
-        "query_function_call_error_1",
-        "Query.function_call error 1: ReadRPC success, NEAR RPC error"
-    ).unwrap();
+    pub(crate) static ref OPTIMISTIC_UPDATING: OptimisticUpdating = OptimisticUpdating::new();
```

Comment:
This is the metric I want to expose to add the panel/chart in grafana, but it's not really a metric. Please expose it properly